# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jsaw/epplets-extra16/e-cpu2.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- E-CPU2v0.2/COMPILE.orig-rock-maint	2003-08-29 08:34:25.000000000 +0200
+++ E-CPU2v0.2/COMPILE	2003-08-29 09:02:11.000000000 +0200
@@ -24,8 +24,8 @@
 echo $CC $CFLAGS `imlib-config --cflags` -c $EPPLET.c -o $EPPLET.o
 $CC $CFLAGS `imlib-config --cflags` -c $EPPLET.c -o $EPPLET.o
 
-echo $CC $CFLAGS `imlib-config --cflags` -o $EPPLET.epplet $EPPLET.o `imlib-config --libs` -lepplet
-$CC $CFLAGS `imlib-config --cflags` -o $EPPLET.epplet $EPPLET.o  `imlib-config --libs` -lepplet
+echo $CC $CFLAGS `imlib-config --cflags` -o $EPPLET.epplet $EPPLET.o `imlib-config --libs` -L$E16/lib -lepplet
+$CC $CFLAGS `imlib-config --cflags` -o $EPPLET.epplet $EPPLET.o  `imlib-config --libs` -L$E16/lib -lepplet
 
 # Install E-NetFlame to the Enlightenment /bin dir
 echo install -m 755 ./$EPPLET.epplet $EBIN/$EPPLET.epplet
